source/math/FloatTransform3D.ooc

```diff
@@ -206,10 +207,13 @@ FloatTransform3D: cover {
 		this o == other o &&
 		this p == other p
 	}
-	transformProjected: func (point: FloatPoint2D, focalLength: Float) -> FloatPoint2D {
+	transformProjected: func ~FloatPoint2D (point: FloatPoint2D, focalLength: Float) -> FloatPoint2D {
```

Comment:
What does transformProjected do? It seams to transform it before it projects it? Is the name the right one?